server/src/main/java/com/objectcomputing/checkins/services/guild/member/GuildMemberRepository.java

```diff
@@ -22,6 +25,13 @@
 
     Optional<GuildMember> findByGuildidAndMemberid(UUID guildMemberid, UUID memberId);
```

Comment:
is this needed or query can be used for this as well ?

Reply:
I think this serves a different role than the search and should not be rolled up into it, but many of the others on this class safely can, I think.